feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go

```diff
@@ -244,8 +247,6 @@ func BuildBenchmarkingConfig(t *testing.T) *oc.Root {
 		// Configure ISIS AfiSafi enable flag at the global level
 		if deviations.MissingIsisInterfaceAfiSafiEnable(dut) {
 			isisIntf.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV4, oc.IsisTypes_SAFI_TYPE_UNICAST).Enabled = ygot.Bool(true)
-		} else {
-			isisIntfLevelAfi.Enabled = ygot.Bool(true)
```

Comment:
Isn't the above deviation (`deviations.MissingIsisInterfaceAfiSafiEnable(dut)`) supposed to be used when this line is not supported?

Reply:
Done!